code/game/objects/obj_defense.dm

```diff
@@ -47,7 +47,8 @@
 	..()
 	var/tforce = 0
 	if(ismob(AM))
-		tforce = 10
+		var/mob/M = AM
+		tforce = M.throwforce
```

Comment:
when the other change is done you don't need to check what kind of thing it is anymore, just check `AM.throwforce`